source/gcp/supported-services/cloud_audit_logs.rst

```diff
@@ -7,34 +7,76 @@
 Audited resources
 =================
 
-Google Cloud maintains three `audit logs <https://cloud.google.com/monitoring/api/resources#tag_audited_resource>`__ for each Google Cloud project, folder, and organization: **Admin Activity**, **Data Access**, and **System Event**. Wazuh supports collecting those logs by using the **gcp-pubsub** module. Details on how to configure the module can be found :ref:`gcp-pubsub configuration reference <gcp-pubsub>`.
+Google audit logs
+-----------------
 
-.. thumbnail:: ../../images/gcp/gcp-overview.png
+Google Cloud maintains four `audit logs <https://cloud.google.com/logging/docs/audit>`__ for each Google Cloud project, folder, and organization: **Admin Activity**, **Data Access**, **System Event**, and **Policy Denied**.
```

Comment:
```suggestion
Google Cloud provides four types of `audit logs <https://cloud.google.com/logging/docs/audit>`__ for each Google Cloud project, folder, and organization:
```

Reply:
it is redundant to list the log types and then to list them again providing more detail